google/cloud/bigquery/schema.py

```diff
@@ -560,6 +560,59 @@ def to_api_repr(self) -> dict:
         return answer
 
 
+class ForeignTypeInfo:
+    """Metadata about the foreign data type definition such as the system in which the
+    type is defined.
+    Args:
```

Comment:
Nit: needs a blank line to separate the sections 

```suggestion

    Args:
```

Reply:
Resolved.